vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { getViteConfig } from "astro/config";
+
+export default getViteConfig({
+  test: {
+    // Vitest configuration options
+    include: ["./**/*.test.ts", "./**/*.test.tsx", "./**/*.tests.ts"],
```

Comment:
Refactor (non-blocking): Most projects I've worked on [co-locate tests](https://kentcdodds.com/blog/colocation). I'd recommend we do that in this project as well, but I'll let others chime in.

Reply:
I don't understand co-locate yet. I'll take a look at this tomorrow. Thanks, @nickytonline!

Reply:
Yeah, I like the idea of co located the tests. 

@adiati98 
Kent's article gives a good explanation for co locating tests. 
But also, here is an example of Nick doing this for the open sauced app repo
https://github.com/open-sauced/app/pull/2275/files

Reply:
I took a look at the OpenSauced's app and kind of understand that we want to put the files that we want to test in the same folder as the tests files.

In this PR, do we want to rename the `test` to `lib/utils` folder and rename the file to be `.test.ts` ?

Reply:
Since this is just a "test" test file it's probably fine like this for now, but we can probably move it into the root of the `src` folder in this PR. Going forward, if tests need to be written, they should reside in the same folder as the code.

We can add that to the contributing guidelines here or in a follow up PR.

Reply:
I've moved the `test` folder to the root of the `src` folder.

And I've updated the README to add the test folder into the project structure. Please let me know if this what you meant by adding to the guidelines.

Thanks, @nickytonline!